test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs

```diff
@@ -118,18 +119,20 @@ public void ShortFrameworkNamesAreCorrect(string longName, string version, strin
 
         [Theory]
         [InlineData("dnx46", "dotnet,dnx46", "dnx46")]
+
+        // Portable frameworks should use the old matching system based on scoring to ensure the best PCL is chosen
+        [InlineData(".NETPortable,Version=v4.0,Profile=Profile6", "portable-net40+sl4+win8,portable-net40+win8", "portable-net40+win8")]
```

Comment:
Before this change, `portable-net40+sl4+win8` would be taken because the Identifier (`.NETPortable`) matched and `portable-net40+sl4+win8` was a Portable 4.5 profile, whereas `portable-net40+win8` was a Portable 4.0 profile or something (I don't remember the specific version numbers :))